packages/docs/components/demos/NodeDemo.tsx

```diff
@@ -1,14 +1,8 @@
 import { DataStory } from '@data-story/ui'
-import { Application, nodes, coreNodeProvider, core } from '@data-story/core';
+import { core, nodes } from '@data-story/core';
 import { MockJSClient } from '../splash/MockJSClient';
 
-export default ({ nodeName }: { nodeName: string}) => {
-  const app = new Application();
-
-  app.register(coreNodeProvider);
-
-  app.boot();
-
+export default ({ nodeName }: {nodeName: string}) => {
   const diagram = core.getDiagramBuilder()
```

Comment:
Is getDiagramBuilder called on a promise here? Compare with comment on `core` export above.

Reply:
Since the core in `core.ts` is not a `Promise<Application>`, `getDiagramBuilder` does not depend on any asynchronous methods, so it won't be affected.